saleor/tax/models.py

```diff
@@ -80,6 +81,9 @@ class TaxConfiguration(ModelWithMetadata):
         Channel, related_name="tax_configuration", on_delete=models.CASCADE
     )
     charge_taxes = models.BooleanField(default=True)
+    tax_calculation_strategy = models.CharField(
```

Comment:
Do you expect that in the future we will have more options here? If not, wouldn't be simpler to make just boolean field like `use_flat_rate`?

Reply:
Also I think, that it would be usefull to have here a default value (if we are able to determine a proper value in migrations)

Reply:
Null is the default value- it is allowed that taxes are not calculated, so no method is selected. I would keep the enum, is more readable to me.

Reply:
`use_flat_rate` does not explain what happens when the option is `false`. Disabling taxes means not charging tax, it's not the same as not calculating tax. If you provide gross prices, we still need to determine a net price, and we need to know how to do it (app or flat rate).

Reply:
I solved the NON_TAXES value by making the enum optional in API.